components/query_filter/utils.h

```diff
@@ -3,13 +3,17 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_BROWSER_NET_BRAVE_QUERY_FILTER_H_
-#define BRAVE_BROWSER_NET_BRAVE_QUERY_FILTER_H_
+#ifndef BRAVE_COMPONENTS_QUERY_FILTER_UTILS_H_
+#define BRAVE_COMPONENTS_QUERY_FILTER_UTILS_H_
 
+#include <string>
 #include "third_party/abseil-cpp/absl/types/optional.h"
```

Comment:
there should be a blank space between system headers and chromium headers https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Its too bad the linter can't catch this :(

Reply:
the linter absolutely should be able to catch this.  @ShivanKaul @goodov i know we made improvements in Brave's linting of brave-core code before.  Can we add this too?

Reply:
that would be great. In any case I fixed this.